commons/pac-api-commons/src/main/java/com/tmobile/pacman/api/commons/utils/CommonUtils.java

```diff
@@ -131,6 +128,16 @@ public static LinkedHashMap<String, Object> flatNestedLinkedHashMap(String notat
 		return flatNestedMap;
 	}
 
+	public static String joinListElement(List<String> list) {
+		if(CollectionUtils.isEmpty(list)){
+			return "";
+		}
+		return String.join(",", list
+				.stream()
+				.map(name -> ("\"" + name + "\""))
+				.collect(Collectors.toList()));
+	}

```

Comment:
1. do you see any reuse cases for this method so it can stay in common? If not - it should be in the class where it is used.
2. Naming is not exactly correct - this method is not only joining list, but it also alter its members before they are turned into a string. Should be clear what it's doing from the name.

Reply:
Moved this method to class with appropriate generic name.